uo_sys_framework/Core/CfgUO/RespawnTemplates.hpp

```diff
@@ -1,10 +1,10 @@
 class MGVAR(RespawnTemplates) {
     respawnTemplatesNone[] = {};
-    respawnTemplatesUnlimited[] = {QEGVAR(RTemplatesS,JoinGroup),QEGVAR(RTemplatesS,SafeStart),QEGVAR(RTemplatesS,ACRE),QEGVAR(RTemplatesS,Gear),QEGVAR(RTemplatesS,CoverMap),QEGVAR(RTemplatesS,MapAndCompassRemover)};
```

Comment:
Rather than removing the option, we should add information to the tooltip and make it off by default,

Reply:
It is not removed. It is not set as default